utilities/ft_checkdata.m

```diff
@@ -1456,6 +1456,10 @@
     % reshape the first dimension
     tmp = getsubfield(data, fn{k});
     data.(fn{k}) = reshape(tmp, [data.dim dimsiz(2:end)]);
+  elseif startsWith(dimord, '{pos}')
```

Comment:
should we then also plan for `{pos_pos}`, which would be a 6D cell array?

Reply:
btw, reshaping the cell-array should indeed be fine

Reply:
We could build in support for '{pos_pos}', let's see what can of worms will be opened

Reply:
OK, I think it's done.